platform/android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/offline/OfflineRegion.java

```diff
@@ -429,6 +429,53 @@ public void run() {
     }
   }
 
+  /**
+   * Same as the method above but skipping database file packing for performance reasons.
+   * <p>
+   * This method is useful optimization e.g. when several regions should be deleted in a row.
```

Comment:
Would be great to highlight here that the database file size will not shrink and you need to call `packDatabase` to do it later (and link the method like above).